errorpage.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+    <head>
```

Comment:
this `<head>` information is already part of the jekyll layouts. I have added a `blank` layout with all of the head information and the footer. Check the other HTML files on the site for the code to use to implement this header